scripts/exregional_process_bufr.sh

```diff
@@ -212,7 +228,7 @@ esac
 #-----------------------------------------------------------------------
 
 run_lightning=false
-obs_file=${obspath_tmp}/${obsfileprefix}.t${HH}z.lghtng.tm00.bufr_d
+obs_file=${obspath_tmp}/${obsfileprefix}.t${HH}${SUBH}z.lghtng.tm00.bufr_d
```

Comment:
where to set SUBH?

Reply:
It is defined inside this script.  Check around Line 200 of the new file.